### ISSUE TITLE
False Positive? `word-break: break-word` triggers an error

### ISSUE THREAD

**Issue by erwanjugand** [ryo-manba/stylelint-plugin-use-baseline#24]
`word-break: break-word;` return a error :

```
Value "break-word" of property "word-break" is not a widely available baseline feature. (plugin/use-baseline)
```

[word-break](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break) is marked as `widely `supported in the MDN documentation.

However, the value `break-word` is listed as **deprecated** — is that the reason why it triggers an error?

**Comment by ryo-manba** [comment]
Thanks @rviscomi, that makes sense!

You're right, the message could be clearer.
Instead of `"not a widely available baseline feature"`, something like `"discouraged feature"` might be more accurate in this case.

If you have any suggestions, feel free to open an issue.

**Comment by ryo-manba** [comment]
I've created the issue.
- https://github.com/ryo-manba/stylelint-plugin-use-baseline/issues/64
